src/components/fields/location-field/location-modal/location-picker/location-picker.styles.ts

```diff
@@ -74,9 +72,9 @@ export const ButtonLocation = styled.button`
 	border: 0;
 	border-radius: 50%;
 	padding: 0;
-	background: ${V2_Color.Neutral[8]};
+	background: ${Colour.bg};
 	cursor: pointer;
-	box-shadow: 0 0.125rem 0.25rem ${V2_Color.Neutral[1]}66;
+	box-shadow: 0 0.125rem 0.25rem ${Colour["bg-inverse"]}66;
```

Comment:
there's no semantic equivalent for box shadow at the moment, suggest to use the primitive directly

```
box-shadow: 0 0 4px rgb(from ${Colour.Primitive["neutral-20"]} r g b / 40%);
```